src/microbe_stage/Microbe.Interior.cs

```diff
@@ -622,6 +675,10 @@ private void HandleCompoundAbsorbing(float delta)
         cloudSystem!.AbsorbCompounds(GlobalTransform.origin, grabRadius, Compounds,
             TotalAbsorbedCompounds, delta, Membrane.Type.ResourceAbsorptionFactor);
 
+        // Cells with jets aren't affected by mucilage
+        slowedBySlime = cloudSystem.AmountAvailable(mucilage, GlobalTransform.origin, 1.0f) >
+            Constants.COMPOUND_DENSITY_CATEGORY_FAIR_AMOUNT && SlimeJets.Count < 1;
```

Comment:
So if I understand well, the amount of slime does not effect how much you're slowed? A bit unintuitive to me, but this is fine code-wise.